reinforcement_learning/rl_unity_ray/rl_unity_ray.ipynb

```diff
@@ -7,7 +7,8 @@
     "# Unity3D Game with Amazon SageMaker RL\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #6.    image_name = "462105765813.dkr.ecr.{region}.amazonaws.com/sagemaker-rl-ray-container:ray-{ray_version}-tf-{cpu_or_gpu}-{suffix}".format(
</pre><p>Can this be fetched using <code>sagemaker.image_uris.retrieve()</code>	rather than hard-coded?  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3237/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>

Reply:
Unfortunately not, this is a special container created by the RL team themselves. sagemaker.image_uris.retrieve(), to my knowledge, only fetches DLC and 1P containers